docs/quickstart/deploy-airbyte.md

```diff
@@ -1,6 +1,6 @@
-# Deploy Airbyte
+# Deploy Airbyte Open-Source
```

Comment:
Are we comitting to calling it `Airbyte Open-Source` or are we calling it `Airbyte Community`?